react-gbajs/src/emulator/video/software.js

```diff
@@ -814,7 +862,7 @@ GameBoyAdvanceSoftwareRenderer.prototype.clear = function(mmu) {
 		this.drawBackdrop
 	];
 
-	this,objwinActive = false;
+	this.objwinActive = false;
```

Comment:
this is just a lil typo I noticed that is in the gbajs code, so I fixed it. No idea what implications this has :)

Reply:
😱
I debugged it just now, and it looks like that the behavior will continue to be the same.